Documentation/help_files/mfa_help.php

```diff
@@ -0,0 +1,319 @@
+                            
+                            <p><?php echo xlt("Web based methods use Authentication-as-a-service and deliver OTP without need for tokens");?>.
+                            
+                            <p><?php echo xlt("Hard copy OTP is used for online banking in some countries");?>.
+                        </div>
+                    </div>
+                    
+                    <p><?php echo xlt("OpenEMR offers 2 methods of 2FA");?>.
+                        <ul>
+                            <li><?php echo xlt("TOTP - Time-Based One-Time Password"); ?></li>
+                            <li><?php echo xlt("FIDO U2F - Universal 2nd Factor from the Fast IDentity Online alliance"); ?></li>
+                        </ul>
+                        
+                    <p><?php echo xlt("The help file is divided into the following sections");?>:
+                        
+                    <ul id="top_section">
+                        <li><a href="#section1"><?php echo xlt("Setting up 2FA");?></a></li>
+                        <li><a href="#section2"><?php echo xlt("Advantages and Disadvantages of TOTP and U2F");?></a></li>
+                        <li><a href="#section3"><?php echo xlt("Using 2FA");?></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class= "row" id="section1">
+                <div class="col-sm-12">
+                    <h4 class="oe-help-heading"><?php echo xlt("Setting up 2FA"); ?><a href="#top_section"><i class="fa fa-arrow-circle-up oe-pull-away oe-help-redirect" aria-hidden="true"></i></a></h4>
+                    <p><?php echo xlt("Clicking on the Miscellaneous > MFA Management menu item or  User Name > MFA Management menu item brings you to the landing page for managing the multi factor autentication page for that user");?>.
+                    
+                    <p><i class="fa fa-exclamation-circle oe-text-orange"  aria-hidden="true"></i> <?php echo xlt("It is important to realize that this manages the MFA for the logged in user");?>.
+                    
+                    <p><?php echo xlt("Therefore each user has to individually activate this feature");?>.
+                    
+                    <p><?php echo xlt("There are two sections that are visible");?>:
+                        <ul>
+                            <li><?php echo xlt("Current Authentication Method for the logged in user"); ?></li>
+                            <li><?php echo xlt("Select/Add Authentication Method for the logged in user"); ?></li>
+                        </ul>
+                    
+                    <p><?php echo xlt("The Current Authentication Method lists all the methods that are active");?>.
+                    
+                    <p><?php echo xlt("OpenEMR allows one TOTP and one U2F method per user");?>.
```

Comment:
OpenEMR allows:
one TOTP per user
multiple U2F per user

Reply:
Thanks for the info. I had to reverse engineer based on my reading of the code :)

Reply:
What is the purpose of  allowing multiple U2F keys ?

Reply:
@sunsetsystems wrote the code for U2F and appears it cycles through all the keys for a match. Guessing several use cases. Maybe one is:
1. A main key and also a backup key

Reply:
Read my cons for U2F keys, can be forgotten on the computer used to login.  Having multiple keys would likely weaken security. Instead user's key should to be deleted by admin and then the user be  allowed to register a new key